src/seedu/addressbook/AddressBook.java

```diff
@@ -1146,12 +1182,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param inputString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
```

Comment:
Correct! Comments should be updated to match changes in code logic.

Note that you probably should remove the mention of "sign" from this comment though.